runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -152,6 +153,8 @@ public void write(IToken token, Object data) throws OverwriteException {
                 // to validate the state of the write, we know that it didn't went through.
                 if (re instanceof OverwriteException) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this conditional structure or edit its code blocks so that they're not all the same. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3923)